src/Writer/Renderer/Entry/Rss.php

```diff
@@ -218,7 +220,7 @@ protected function _setEnclosure(DOMDocument $dom, DOMElement $root)
         }
         $enclosure = $this->dom->createElement('enclosure');
         $enclosure->setAttribute('type', $data['type']);
-        $enclosure->setAttribute('length', $data['length']);
+        $enclosure->setAttribute('length', (string) $data['length']);
```

Comment:
Ok, this will likely not fly without a major release 😬

Reply:
L201:L220 assert the value is an integer - why would this be a BC break?

Reply:
Ah, confused this with a test, sorry.